### PR TITLE
Commerce shipment

### PR DESCRIPTION
**Pull request by justinchoi001**
new PR based on feedback and comments

### CODE REVIEW

File: docs/commerce/2.x/en/orders-and-fulfillment/managing-shipments/introduction-to-shipments.md [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+![Shipments Menu - Finish Processing](./introduction-to-shipments/images/05.png)
+
+Once all the items and the sourcing warehouse chosen, the order is ready to ship. Here, you can view additional details such as the carrier, tracking number, and estimated delivery date. Clicking the _Ship_ button advances the order's process.
+
+![Shipments Menu - Ready to Ship](./introduction-to-shipments/images/06.png)
+
+When delivery has been confirmed, the shipment process marked delivered and the order is marked completed.
+
+![Shipments Menu - Ready to Ship](./introduction-to-shipments/images/07.png)
+
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/managing-shipments/introduction-to-shipments.md:20]: This section is still structured inconsistently with the commerce 2.0 section - please make the organization consistent.. Please schedule a call with me to discuss if you're not sure what I mean.

File: docs/commerce/2.x/en/orders-and-fulfillment/orders-menu.md [hunk 2]
```diff
@@ -1,28 +1,90 @@
 # Orders Menu
 
-All orders in Liferay Commerce can be viewed and managed by an administrator in the _Orders_ menu. Here, orders are displayed in the _Open_, _Pending_, and _Transmitted_ tabs according to the order’s progress through the [order life cycle](./order-life-cycle.md).
+All orders in Liferay Commerce can be viewed and managed by an administrator in the _Orders_ menu. To access the _Orders_ menu, navigate to the _Control Panel_ &rarr; _Commerce_ &rarr; _Orders_.
+
+There are five tabs at the top: _All_, _Open_, _Pending_, Processing_, and _Completed_.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/orders-menu.md:5]: `Processing_` formatting error.

**Comment by jrhoun** [comment]
See my remarks. Thanks!

File: docs/commerce/2.x/en/orders-and-fulfillment/orders-menu.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/orders-menu.md:5]: Fixed

File: docs/commerce/2.x/en/orders-and-fulfillment/managing-shipments/introduction-to-shipments.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/managing-shipments/introduction-to-shipments.md:20]: I've added section headings.

**Pull request closed without merging**